### PR TITLE
bring up to date

### PR DESCRIPTION
**Pull request by JarbasAl**
for integration with https://github.com/OpenVoiceOS/ovos-backend-client/pull/28

### CODE REVIEW

File: neon_solvers/__init__.py [hunk 1]
```diff
@@ -43,13 +43,15 @@ def __init__(self, bus=None, config=None):
...
-            if plug_name in self.config:
-                try:
-                    config = self.config.get(plug) or {}
-                    self.loaded_modules[plug_name] = plug(config=config)
-                    LOG.info(f"loaded question solver plugin: {plug_name}")
-                except Exception as e:
-                    LOG.exception(f"Failed to load question solver plugin: {plug_name}")
+            config = self.config.get(plug) or {}
+            if not config.get("enabled", True):
+                continue
```
Comment (NeonDaniel) [inline neon_solvers/__init__.py:50]: Can this include a debug log that the plugin is disabled

**Review by NeonDaniel — CHANGES_REQUESTED** [review]

File: neon_solvers/__init__.py [hunk 2]
```diff
@@ -27,32 +27,36 @@
...
 
     def load_plugins(self):
         for plug_name, plug in find_question_solver_plugins().items():
-            if plug_name in self.config:
-                try:
-                    self.loaded_modules[plug_name] = plug()
-                    LOG.info(f"loaded question solver plugin: {plug_name}")
-                except Exception as e:
-                    LOG.exception(f"Failed to load question solver plugin: {plug_name}")
+            config = self.config.get(plug) or {}
```
Comment (NeonDaniel) [inline neon_solvers/__init__.py]: should be `plug_name`, not `plug`?

File: neon_solvers/__init__.py [hunk 3]
```diff
@@ -27,32 +27,36 @@
...
+            if not config.get("enabled", True):
+                continue
+            try:
+                config = self.config.get(plug) or {}
```
Comment (NeonDaniel) [inline neon_solvers/__init__.py]: `config` already defined?

**Comment by JarbasAl** [comment]
replaced by https://github.com/OpenVoiceOS/ovos-plugin-manager/pull/140

**Comment by JarbasAl** [comment]
nvm, the service should stay here, only base class moved, will update PR

**Pull request closed without merging**